test/unit/BCoWPool.t.sol

```diff
@@ -306,31 +255,28 @@ contract BCoWPool_Unit_IsValidSignature is BaseCoWPoolTest {
     bCoWPool.finalize();
   }
 
-  modifier _withTradingEnabled(bytes32 _appData) {
-    bCoWPool.set_appDataHash(keccak256(abi.encode(_appData)));
-    _;
-  }
-
-  modifier _withValidCommitment(GPv2Order.Data memory _order) {
+  modifier happyPath(GPv2Order.Data memory _order) {
+    // happy values
```

Comment:
nitpick: 
I'd either remove the comments or explain _why_ the value is correct and where is set, something along the lines of 'hard-code it to the same value used when deploying the factory in `setUp` so it passes the first check on `isValidSignature`'